lib/puppet/provider/service/systemd.rb

```diff
@@ -25,7 +25,7 @@
   defaultfor :osfamily => :coreos
   defaultfor :operatingsystem => :amazon, :operatingsystemmajrelease => ["2"]
   defaultfor :operatingsystem => :debian, :operatingsystemmajrelease => ["8", "stretch/sid", "9", "buster/sid"]
```

Comment:
It looks like this also misses Debian 10 where it's also the default.

Reply:
This is a good catch! Are we able to continue this thread in https://github.com/puppetlabs/puppet/pull/8593? I've opened the new MR to fix the branch naming convention 😄